native_words.asm

```diff
@@ -5843,39 +5818,34 @@ xt_plus_loop:
                 lda cp
                 sec
                 sbc #1
-                sta tmp2
+                pha             ; lsb
```

Comment:
push saves a couple of bytes and simplifies template copy below

Reply:
This looks good.  Using the tmpx variables needs a LOT of care to make sure any words that are JSRd to don't also use them (or you push/pop them).  It looks like you've removed some of the tmpx usage, which is great.